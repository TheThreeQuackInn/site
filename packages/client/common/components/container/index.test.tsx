import React from 'react';
import {create} from 'react-test-renderer';
import Container from './';

describe('Container', () => {
    it('renders correctly without className', () => {
        const component = create(
            <Container>
                <p>Children</p>
            </Container>,
        );
        expect(component).toMatchSnapshot();
    });

    it('renders correctly with className', () => {
        const component = create(
            <Container className="pb-1">
                <p>Children</p>
            </Container>,
        );
        expect(component).toMatchSnapshot();
    });
});
