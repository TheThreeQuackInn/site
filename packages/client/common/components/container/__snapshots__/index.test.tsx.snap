// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container renders correctly with className 1`] = `
<div
  className="container mx-auto flex pb-1"
>
  <p>
    Children
  </p>
</div>
`;

exports[`Container renders correctly without className 1`] = `
<div
  className="container mx-auto flex"
>
  <p>
    Children
  </p>
</div>
`;
